add_menufunc.js

```diff
@@ -401,6 +401,39 @@ function displayProject(div2c, image, title, subt1, subt2, subt3, iml, tech0, te
   a4chk.textContent = 'See Project';
 }
 
+const form = document.querySelector('form');
+
+function validation() {
+  const mail = document.getElementById('email').value;
```

Comment:
- [ ] The email field of your form is accepting **incorrect email input**. I inputed an incomplete email `se@f` and this was submitted successfully. Kindly ensure that your email field is properly validated against the wrong email format

Reply:
I have fixed this issue now. It's resolved

Reply:
- [ ] Even though my email is in lowercase it still shows as an error. Kindly fix this
![Screenshot from 2021-10-15 20-49-57](https://user-images.githubusercontent.com/62705947/137538006-18a96e48-5da5-4aae-98ca-3af897cf39c0.png)